mk_verificator/utils/__init__.py

```diff
@@ -38,7 +38,18 @@ def get_groups(config):
 
 
 def get_configuration(path_to_test):
+    """global conf is minor, but local conf is major."""
```

Comment:
okies, let's rephrase it:

function returns configuration for environment and for test if it's specified.